src/components/LoansByCategory/QuickFilters/LocationSelector.vue

```diff
@@ -226,6 +226,14 @@ export default {
 				this.$emit('handle-overlay', false);
 			}
 		},
+		closeRegions() {
+			if (this.showRegions) {
+				this.showRegions = false;
+				this.selectedRegion = null;
+				document.documentElement.style.overflow = 'auto';
+				this.$emit('handle-overlay', false);
```

Comment:
Can't we call `toggleRegions` instead?

Reply:
It was calling `toggleRegions` but there is a bug described here; toggling when dont needed: https://kivahq.slack.com/files/U0278NCFQ01/F04FC8U2TKN/screen_recording_2022-12-14_at_1.51.51_pm.mov

Reply:
That link isn't working for me, but my guess based on the original code was that it displayed when outside of the location selector was clicked? Generally this type of logic should be consolidated into one path, for example if the emitted event ever changes, we'd have to remember to update all locations. I'll resolve comment for now since `toggleRegion` is used a bunch in this file, and I don't want to derail this work, but I recommend in the near future thinking of a way to combine these methods. One possible solution would be provide an override flag to the toggle method.